.env

```diff
@@ -0,0 +1 @@
+gist_token = 81b0b2c67a98fcca4827e2c8899db913829c7e04
```

Comment:
can we have something like
`gist_token = <token>`
then each will have to provide his own (that should be documented in the readme too).
Cause i can't commit that this token will always be working

Reply:
Yes, okay. I'll make the change.